docs/vs-code-extension-guide/master.adoc

```diff
@@ -8,21 +8,18 @@ include::topics/templates/document-attributes.adoc[]
 :context: vsc-extension-guide
 :vsc-extension-guide-guide:
 
-= {VscExtensionBookName}
-
-This guide is for engineers, consultants, and others who want to use the {PluginName} for the {ProductName} ({ProductShortName}) to assist with migrating applications.
+= Visual Studio Code Extension Guide
```

Comment:
Just a small comment. I'm pretty sure we adopted sentence case for titles quite a while ago. Not high priority but you should think about checking that.

Reply:
Will check. Probably a repo-wide issue.